examples/src/main/java/io/github/tcdl/examples/RequesterResponderTest.java

```diff
@@ -5,12 +5,14 @@
 /**
  * Created by rdrozdov-tc on 6/15/15.
  */
-public class RequesterResponderTest extends BaseExample {
+public class RequesterResponderTest {
 
     private static final Integer NUMBER_OF_RESPONSES = 1;
 
     final String NAMESPACE = "test:requester-responder-example";
 
+    private MsbTestHelper helper = MsbTestHelper.getInstance();
```

Comment:
Maybe "private MsbTestHelper msbTestHelper = MsbTestHelper.getInstance();" will be better?